README.md

```diff
@@ -65,6 +65,8 @@ Basic configuration options are detailled [here](https://kubernetes.github.io/in
 
 You can find your external MongoDB prerequisites [here](/doc/mongo-prerequisites.md).
 
+> **Caution**: You _CANNOT_ connect multiple NeoLoad Web instances (whether they are in the same namespace or not) to the same MongoDB instance.
```

Comment:
```
> [!CAUTION]
> You _CANNOT_ connect multiple NeoLoad Web instances (whether they are in the same namespace or not) to the same MongoDB instance.
```

Based on [this](https://github.com/orgs/community/discussions/16925)

Reply:
Thank you for the tips, I'll impact the full doc with it